cypress/e2e/mod6_homework.cy.js

```diff
@@ -0,0 +1,60 @@
+const url = 'https://jsonplaceholder.typicode.com/posts'
+const url2 = 'https://jsonplaceholder.typicode.com/posts/1'
```

Comment:
tutaj myślę, że mógłbyś zostawić jeden url ze / na końcu. W testach za to robić coś takiego: url + '1'